src/App.js

```diff
@@ -49,25 +47,21 @@ function App() {
   useEffect(() => {
     window.location.hash = "";
 
-    if (_token) {
+  if (_token) {
+    dispatch({
+      type: "SET_TOKEN",
+      token: _token,
+    });
+    spotify.setAccessToken(_token);
+    spotify.getMe().then((user) => {
```

Comment:
`question`: Why are you using `.then` here? Is there another syntax you could be using here? Could you compare and contrast your available options?